<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RpcFuzzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="IDL_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUT IDL</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>38</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>test</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="output_lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>60</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>test</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="input_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Default Name - Input : </string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="output_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Same IDL Name - Output : </string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="prot_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ncalrpc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ncacn_np</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="EndPoint_lineEdit">
     <property name="geometry">
      <rect>
       <x>198</x>
       <y>90</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>End Point</string>
     </property>
    </widget>
    <widget class="QListWidget" name="methods_listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>147</y>
       <width>381</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="retry_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>489</y>
       <width>381</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retry Compile</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="handle_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Handle</string>
     </property>
    </widget>
    <widget class="QPushButton" name="all_compile_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>442</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>All Compile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="all_run_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>442</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>All Run</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nulldere_checkBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Null Dere</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
